tests/applications.js

```diff
@@ -1,28 +1,134 @@
 import test from 'ava';
+import events from 'events';
+import httpMocks from 'node-mocks-http';
 import config from 'config';
 import _ from 'lodash';
+import uuid from 'uuid';
+import moment from 'moment';
 
 import applications from '../lib/applications';
 import database from '../lib/database';
 import merged from './fixtures/applications/objects/model-merged.js';
 
+const EventEmitter = events.EventEmitter;
+
+/**
+ * Next
+ *
+ * @param {object} value object send to next
+ *
+ * @returns {object} whatever the function received
+ */
+const next = (value) => {
+  return value;
+};
+
+const date = moment().format('YYYY-MM-DD hh:mm');
+
+const body = {
+  'name--text': 'Bar',
+  'live-endpoint--url': 'http://bar.com/live',
+  'updated-endpoint--url': 'http://bar.com/updated',
+  'sunset-endpoint--url': 'http://bar.com/sunset',
+  'client-id': uuid.v4(),
```

Comment:
These aren't fields; they're [generated and just displayed](https://github.com/punchcard-cms/punchcard/blob/master/views/applications/all.html#L19-L40). Should not be included in the test body.